src/kit.ts

```diff
@@ -347,8 +353,8 @@ export async function kitIfCompiler(bin: string, pr?: ProgressReporter): Promise
     const clang_cl_res = clang_cl_regex.exec(fname);
 
     if (gcc_res) {
-        const version = await getCompilerVersion('GCC', bin, pr);
-        if (version === null) {
+        const version = isTrusted ? await getCompilerVersion('GCC', bin, pr) : null;
+        if (isTrusted && version === null) {
```

Comment:
why is the `isTrusted` check needed here? `version` should be null if `isTrusted === false` right?